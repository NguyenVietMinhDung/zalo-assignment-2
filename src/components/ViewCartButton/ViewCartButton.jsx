import React from 'react';
import './ViewCartButton.scss';

const ViewCartButton = () => (
  <div className="view-cart-btn">
    <div className="view-cart-btn__text">
      <div>Xem đơn hàng</div>
      <div>1</div>
      <div>47.000đ</div>
    </div>
  </div>
);

export default ViewCartButton;
