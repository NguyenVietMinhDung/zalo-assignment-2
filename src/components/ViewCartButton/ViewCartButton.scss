@import '../../styles/variables';

.view-cart-btn {
	position: sticky;
	bottom: 0;

	padding: 8px 0;

	cursor: pointer;

	background-color: transparent;

	&__text {
		display: flex;
		flex-direction: row;

		width: 320px;
		max-width: 100vw;
		margin: auto;
		padding: 15px;

		color: white;
		border-radius: 50px;
		background-color: $light-red;

		div {
			flex: 1;

			text-align: center;

			&:first-child {
				text-align: start;
			}

			&:last-child {
				text-align: end;
			}
		}
	}
}
