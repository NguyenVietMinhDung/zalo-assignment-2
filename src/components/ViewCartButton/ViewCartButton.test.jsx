import React from 'react';
import { shallow } from 'enzyme';
import ViewCartButton from './ViewCartButton';

describe('<ViewCartButton />', () => {
  test('renders correctly', () => {
    const component = shallow(<ViewCartButton />);
    expect(component).toMatchSnapshot();
  });
});
