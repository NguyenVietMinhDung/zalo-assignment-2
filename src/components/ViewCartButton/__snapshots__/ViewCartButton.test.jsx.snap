// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewCartButton /> renders correctly 1`] = `
<div
  className="view-cart-btn"
>
  <div
    className="view-cart-btn__text"
  >
    <div>
      Xem đơn hàng
    </div>
    <div>
      1
    </div>
    <div>
      47.000đ
    </div>
  </div>
</div>
`;
